decred/tests/unit/dcr/wire/test_msgpong.py

```diff
@@ -0,0 +1,59 @@
+"""
+Copyright (c) 2020, The Decred developers
+See LICENSE for details.
+
+Based on dcrd MsgPing.
+"""
+
+import random
+
+from decred.dcr.wire import wire
+from decred.dcr.wire.msgpong import MsgPong
+from decred.util.encode import ByteArray
+
+
+def test_PongLatest():
+    """
+    Test the MsgPong API against the latest protocol version.
+    """
+    pver = wire.ProtocolVersion
+
+    nonce = random.randint(0, 0xFFFFFFFFFFFFFFFF)
+    msg = MsgPong(nonce)
+    assert msg.nonce == nonce
+
+    # Ensure the command is expected value.
+    assert msg.command() == "pong"
+
+    # Ensure max payload is expected value for latest protocol version.
+    maxPayload = msg.maxPayloadLength(pver)
+    assert maxPayload == 8
+
+    # Ensure max payload length is not more than MaxMessagePayload.
+    assert maxPayload <= wire.MaxMessagePayload
+
+    # Test encode with latest protocol version.
+    b = msg.btcEncode(pver)
+
+    # Test decode with latest protocol version.
+    reMsg = MsgPong.btcDecode(b, pver)
+
+    # Ensure nonce is the same.
+    assert msg.nonce == reMsg.nonce
```

Comment:
these are tested in test_PongWire.

Reply:
Meh. Good point, but I'm okay it. One is random, one is constant.